src/components/Match/MatchHeader/MatchHeader.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Row, Col } from 'react-flexbox-grid';
+import { transformations, isRadiant } from 'utility';
+import strings from 'lang';
+import Spinner from 'components/Spinner';
+import { IconRadiant, IconDire } from 'components/Icons';
+import styles from './MatchHeader.css';
+
+export default ({ match, loading }) => {
+  if (!loading) {
+    const kills = match.players.reduce(
+      (kills, player) => {
+        if (isRadiant(player.player_slot)) {
+          kills.radiant.push(player.kills);
+        } else {
+          kills.dire.push(player.kills);
+        }
+        return kills;
+      },
+      {
+        radiant: [],
+        dire: [],
+      }
+    );
+
+    return (
+      <header className={styles.header}>
```

Comment:
Consider adding space to support team names (which is something I want to add at some point).  Perhaps just make some empty elements and test layout by putting in fake team names.